VisualStudio.gitignore

```diff
@@ -24,6 +24,8 @@ bld/
 
 # Visual Studio 2015 cache/options directory
 .vs/
+# Uncomment if you have tasks that create the project's static files in wwwroot
```

Comment:
@sayedihashimi is this something we should be calling out? How do you feel about the words here?